match/lib/match/module.rb

```diff
@@ -23,7 +23,7 @@ def self.profile_type_sym(type)
   end
 
   def self.cert_type_sym(type)
-    type = type.to_s
+    type = type.to_s.downcase
```

Comment:
New need.

To determine certificate types to fetch from the portal, I use `Sigh.certificate_types_for_profile_and_platform`, and it returns typed `Spaceship::ConnectAPI::Certificate::CertificateType` with the same values but uppercased.

Reply:
Perhaps explain this "why" in a comment, since it's not immediately clear to the reader :)? e.g.:

```suggestion
    # To determine certificate types to fetch from the portal, we use `Sigh.certificate_types_for_profile_and_platform`, and it returns typed `Spaceship::ConnectAPI::Certificate::CertificateType` with the same values but uppercased, so we downcase them here
    type = type.to_s.downcase
```